Tools/dea_tools/spatial.py

```diff
@@ -1,7 +1,6 @@
 ## dea_spatialtools.py
 '''
-Description: This file contains a set of python functions for conducting 
-spatial analyses on Digital Earth Australia data.
+Tools for spatially manipulating Digital Earth Australia data (vectorising, rasterising, interpolating etc).
```

Comment:
Perhaps "(vectorising, rasterising, interpolating etc)" is not needed to enhance conciseness. One can look at the list of functions available instead and infer that vectorisation, rasterisation and interpolation are available.

Reply:
done!